dubbo-remoting/dubbo-remoting-http12/src/main/java/org/apache/dubbo/remoting/http12/message/LengthFieldStreamingDecoder.java

```diff
@@ -167,8 +160,8 @@ private void processBody() throws IOException {
         requiredLength = lengthFieldOffset + lengthFieldLength;
     }
 
-    protected void invokeListener(InputStream inputStream) {
-        this.listener.onFragmentMessage(dataHeader, inputStream);
+    public void invokeListener(InputStream inputStream) {
```

Comment:
**Security Issue**: Public method 'invokeListener' should not expose its internal InputStream directly, consider copying the data or using a safer access method. <br> **Fix**: Ensure encapsulation of internal streams to prevent unauthorized stream manipulation. <br> **Code Suggestion**: 
 ```
 -    protected void invokeListener(InputStream inputStream) {
 +    public void invokeListener(InputStream inputStream) {
 ```